tests/ssl-opt.sh

```diff
@@ -5308,8 +5308,8 @@ run_test    "DTLS proxy: 3d, min handshake, resumption" \
             -p "$P_PXY drop=5 delay=5 duplicate=5" \
             "$P_SRV dtls=1 hs_timeout=250-10000 tickets=0 auth_mode=none \
              psk=abc123 debug_level=3" \
-            "$P_CLI dtls=1 hs_timeout=250-10000 tickets=0 psk=abc123 \
-             debug_level=3 reconnect=1 read_timeout=1000 max_resend=10 \
+            "$P_CLI dtls=1 hs_timeout=500-10000 tickets=0 psk=abc123 \
```

Comment:
You've changed this timeout (and likewise for one other test case below). I'm ok with the change since it aligns 2.7 with the other stable branches. However please do this in a separate commit.